honeycombio/resource_slo.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 
+	"github.com/honeycombio/terraform-provider-honeycombio/client"
```

Comment:
this package is already imported as `honeycombio`: a bit odd but is standard in the SDKv2 code so it'd be good to stick with it